src/explore-education-statistics-admin/src/components/form/FormEditor.tsx

```diff
@@ -204,6 +204,17 @@ const FormEditor = ({
       table: {
         contentToolbar: tableContentToolbar,
       },
+      link: {
+        decorators: {
+          addDataGlossaryAttributeToGlossaryLinks: {
+            mode: 'automatic',
+            callback: (url: string) =>
+              url.startsWith(process.env.PUBLIC_URL) &&
+              url.match(/\/glossary#[a-zA-Z-0-9-]+$/),
+            attributes: { 'data-glossary': '' },
```

Comment:
Unfortunately, there doesn't seem to be a way to pass in the slug (or other identifier for a glossary entry) to the attribute using a link decorator. It seems we'll need to use a plugin to do that.

Reply:
Ooh that's annoying, but yeah I checked it out too and I agree.

Reply:
This is unfortunately because I think this means that, when this feature developed further, we'll either have to continue to parse the URL to determine the glossary entry to be rendered, or we'll have to migrate existing links to include some identifier to the glossary entry into the `data-glossary` attribute.